config/database.yml

```diff
@@ -6,38 +6,24 @@ default: &default
   username: postgres
   password: password
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
 
 development:
   primary:
     <<: *default
     timeout: 5000
   partners:
     <<: *default
-    username: postgres
-    password: password
     database: partner_dev
     migrations_paths: db/partners_migrate
     timeout: 5000
 
-staging:
-  primary:
-    <<: *default
-    url: <%= ENV['DATABASE_URL'] %>
-  partners:
-    <<: *default
-    url: <%= ENV['HEROKU_POSTGRESQL_ROSE_URL'] %>
-    migrations_paths: db/partners_migrate
-
 test:
   primary:
     <<: *default
     database: diaper_test
     timeout: 5000
   partners:
     <<: *default
-    username: postgres
-    password: password
```

Comment:
Hmm, I think this might have been an accident? Did you need to change the database.yml to get the app running?

Reply:
Ahh, it must be this step:
> You can also copy the existing file called database.yml.example as an example and just change the credentials.

This is actually outdated (we should update the README, I'll accept a PR for that too). Could you remove these changes from the PR?

Reply:
How do I remove specific changes from a PR?

Reply:
You can manually add them back in and then make another commit to bring them back.